data_preparation/DATA-CARD-TEMPLATE.md

```diff
@@ -0,0 +1,49 @@
+# Title
+
+## Short description
+
+## Link
+Include the paper link and citation if the data is documented in a paper.
```

Comment:
@krsnapaudel citation is already included in "how to cite".  We can consider "include the link to the data website or paper"

Reply:
Having the actual citation would help us in the long term to avoid looking for each one when making the dataset report or when citing the work in the paper.

Reply:
Link and citations are two different things. Whenever expected citation is specified, it should go to "How to cite".